modules/serverless-autoscaling-maxscale-kn.adoc

```diff
@@ -4,26 +4,26 @@
 
 :_content-type: PROCEDURE
 [id="serverless-autoscaling-maxscale-kn_{context}"]
-= Setting the maxScale annotation by using the Knative CLI
+= Setting the scale-max annotation by using the Knative CLI
```

Comment:
The annotation name is `max-scale` while CLI takes `scale-max` flag.
So here should say `max-scale`.